### ISSUE TITLE
No Swift-3.0 Branch

### ISSUE THREAD

**Issue by mahlon-gumbs** [Jpadilla1/react-native-ios-charts#60]
Hi there.

I was trying to install this module according to your readme and got the following error:

```
Pre-downloading: `Charts` from `https://github.com/danielgindi/Charts.git`, branch `Swift-3.0`

[!] Error installing Charts
[!] /usr/bin/git clone https://github.com/danielgindi/Charts.git /var/folders/nz/__btsnk12tbbqlmk1_gkr0kc0000gn/T/d20160921-14615-uc3mfs --template= --single-branch --depth 1 --branch Swift-3.0

Cloning into '/var/folders/nz/__btsnk12tbbqlmk1_gkr0kc0000gn/T/d20160921-14615-uc3mfs'...
warning: Could not find remote branch Swift-3.0 to clone.
fatal: Remote branch Swift-3.0 not found in upstream origin
```

I've tried replacing that line with one of the other branches found in the Chards repo but I get various errors no matter what I choose. Any chance you can verify this podfile entry?

**Comment by laszlolm** [comment]
@mahlon-gumbs replace it with the `master` branch. By their current standing they support Swift 3 on that branch

**Comment by mahlon-gumbs** [comment]
Hmmm... thanks @noxowe. I'll try that and see if I can at least get things working. The issue there, however, is that I wouldn't be able to check that into our repo for other developers since master will continue to change. Since we don't check in modules or Pods, other developers may wind up with different versions of this. :-(

**Comment by lammertw** [comment]
The `master` branch doesn't work. At the time of writing the `Chart2.2.5-Swift3.0` does appear to work.

**Comment by mahlon-gumbs** [comment]
@lammertw Thanks. Finally got a chance to cleanup my project and got this working with the `Chart2.2.5-Swift3.0` branch you mentioned. I had tried that branch in the past but kept getting hundreds of errors (literally) as Xcode tried the swift conversion.

Turns out...for now, that I had to add the files to the project root rather than have them in a group. I'll try to clean that up at a later time, but at least I can render charts now.
